packages/rscontext/rscontext/rs_context.py

```diff
@@ -333,9 +333,19 @@ def rs_context(huc: str, landfire_dir: str, ownership: str, fair_market: str, ec
     # National Wetland Inventory
     nwi_gpkg = os.path.join(output_folder, LayerTypes['NATIONAL_WETLANDS'].rel_path)
     log.info(f'Processing National Wetland Inventory. Output: {nwi_gpkg}')
-    national_wetlands_inventory(huc, download_folder, nhd['WBDHU10'], nwi_gpkg, cfg.OUTPUT_EPSG)
-
-    project.add_project_geopackage(datasets, LayerTypes['NATIONAL_WETLANDS'])
+    nwi_attempts = 0
+    while nwi_attempts < 3:
+        try:
+            # Download the NWI data
+            national_wetlands_inventory(huc, download_folder, nhd['WBDHU10'], nwi_gpkg, cfg.OUTPUT_EPSG)
+            project.add_project_geopackage(datasets, LayerTypes['NATIONAL_WETLANDS'])
```

Comment:
I think the issue here is that the `download_folder` is actually on the shared drive so that may be part of the issue we're seeing with collisions. 

You can get around this by putting it in the `scratch_folder` instead which should keep it namespaced to this run and then cleaned up after